go-controller/pkg/ovn/controller/services/services_controller_test.go

```diff
@@ -535,6 +622,198 @@ func TestServiceCreateReject(t *testing.T) {
 	controller.syncServices(ns + "/" + serviceName)
 }
 
+// A service can mutate its ports, we need to be sure we don´t left dangling ports
+func TestUpdateServiceEndpointsToHost(t *testing.T) {
```

Comment:
TODO ideal for newcomers, this must be table tests, we are only testing the happy path that is that we know it works